src/common/types/api.js

```diff
@@ -32,6 +32,16 @@ type FetchOptions = {
     refreshCache?: boolean,
 };
 
+type PayloadType = StringAnyMap | Array<StringAnyMap>;
+
+type FetchData = {
+    data: PayloadType,
```

Comment:
This type declaration is a little confusing, as fetch/get calls don't typically have payloads outside of the query string.